index.js

```diff
@@ -14,17 +14,20 @@ var persistState = function (paths, config) {
   var key = config.key || 'storeon'
 
   return function (store) {
+    var initialized = false
+
     store.on('@init', function () {
+      initialized = true
+
       try {
         var savedState = localStorage.getItem(key)
         if (savedState !== null) {
           return JSON.parse(savedState)
         }
       } catch (err) { }
     })
-    store.on('@dispatch', function (state, data) {
-      var event = data[0]
```

Comment:
These lines don't seem necessary anymore, as they will never be executed - please correct me if I'm wrong

Reply:
You are right. This is unnecessary.